BE/OwnershipAndControl/Executives.rdf

```diff
@@ -256,171 +307,93 @@
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/board-of-directors.html</fibo-fnd-utl-av:definitionOrigin>
+		<skos:definition>group of people comprising the governing body of an organization that has the authority to set organizational strategy and policies as well as to select and, to some degree manage, leadership</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>For a public, for profit organization, its members (directors) are elected normally by the subscribers (stockholders) of the firm (generally at an annual general meeting) to govern the firm and look after the subscribers&apos; interests. In the case of a not-for-profit organization, the board ensures the organization is acting in line with its mission. The board has the ultimate decision-making authority and, in general, is empowered to (1) set the company&apos;s policy, objectives, and overall direction, (2) adopt bylaws, (3) name members of the advisory, executive, finance, and other committees, (4) hire, monitor, evaluate, and fire the managing director and senior executives, (5) determine and pay the dividend, and (6) issue additional shares. Though all its members might not be engaged in the company&apos;s day-to-day operations, the entire board is held liable (under the doctrine of collective responsibility) for the consequences of the firm&apos;s policies, actions, and failures to act. Members of the board usually include senior-most executives (called inside directors or executive directors) as well as experts or respected persons chosen from the wider community (called outside directors or non-executive directors).</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-be-oac-exec;Bylaw">
+	<owl:Class rdf:about="&fibo-be-oac-exec;Bylaws">
```

Comment:
Why plural? What if there are 20 Bylaws - will they all be grouped into a single resource?

Reply:
The concept is used in the plural in the US - not sure about elsewhere, but the legal document (in the singular) that incorporates the various definitions and guidelines is called "bylaws" in the US - I have the Sandpiper document somewhere that does that, following a template our attorneys provided.